config/webpack.target.mobile.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const path = require('path')
+const webpack = require('webpack')
+const merge = require('webpack-merge')
+const {filename, production} = require('./webpack.vars')
+
+module.exports = merge(
+  require('./webpack.config.base.js'),
+  production ? require('./webpack.config.prod') : {},
```

Comment:
Why don't you add ` './webpack.config.dev'` as in [./config/webpack.target.split.js](./config/webpack.target.split.js)?